carsus_env3.yml

```diff
@@ -50,13 +50,10 @@ dependencies:
   - pandoc
 
   # Test/Coverage
-  - pytest=4
+  - pytest =4
   - pytest-cov
   - pytest-astropy
-
-  # ChiantiPy
-  - scipy
-  - ipyparallel
+  - pytest-arraydiff >=0.5
```

Comment:
Would it not be better to specify the channel name next to the dependency? Something like [this](https://stackoverflow.com/a/47887331/11974464)?

Reply:
yes, didn't remember this crazy syntax. Thanks